.changeset/gentle-toes-play.md

```diff
@@ -0,0 +1,5 @@
+---
+"@fuel-ts/account": patch
+---
+
+feat: added method to duplicate predicate
```

Comment:
We can have this ready and approved, but let's not merge it just yet. This comment is to prevent an accidental merge.